src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -220,7 +239,72 @@ public String getRemote() {
         }
     }
 
-    protected abstract void doRetrieve(SCMHeadObserver observer, TaskListener listener, GHRepository repo) throws IOException, InterruptedException;
+    protected void doRetrieve(SCMHeadObserver observer, TaskListener listener, GHRepository repo) throws IOException, InterruptedException {
```

Comment:
Could be `private`, and for that matter could just be inlined into `retrieve` since that is the only call.